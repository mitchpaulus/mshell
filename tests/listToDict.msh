[
    ["my key" 10]
    ["my key 2" 20]
    ["my key 2" 30]
]
(:0:)
(:1:)
listToDict str wl
