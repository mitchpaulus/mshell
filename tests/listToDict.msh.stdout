{"my key": {"type": "Integer", "value": 10}, "my key 2": {"type": "Integer", "value": 30}}
